models/topics.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../db/connection')
+
+exports.fetchTopics = () => {
+    return db.query('SELECT * FROM topics')
+        .then((result) => {
+            const topics = result.rows
+            if (topics.length === 0) {
+                const error = new Error('Topics not found')
+                error.statusCode = 404
+                throw error
+            }
+            return topics
+        })
+        .catch((err) => {
```

Comment:
we shouldn't need to have a catch in our model - if an error does occur here the controller's catch will grab that and deal with it 😊